tests/pending/neg/cc-depfun.scala

```diff
@@ -0,0 +1,14 @@
+import language.experimental.captureChecking
```

Comment:
This inconsistency can be reproduced before this PR. It seems that this is a result of `expandAbbreviations`:
https://github.com/lampepfl/dotty/blob/9ed9839e533db22c69bdb1e552ef6eeaacecc333/compiler/src/dotty/tools/dotc/cc/Setup.scala#L286-L296